crates/biome_fs/src/fs/os.rs

```diff
@@ -57,12 +57,27 @@ impl FileSystem for OsFileSystem {
     fn get_changed_files(&self, base: &str) -> io::Result<Vec<String>> {
         let output = Command::new("git")
             .arg("diff")
-            .arg("--name-only")
+            .arg("--name-status")
```

Comment:
We could use `--diff-filter`? For example `git diff --name-only -ddif-filter=d` displays changed files, excluding deleted ones.

Reply:
Damn, git is so full of stuff

Reply:
@Conaclos please check again